ext/opentelemetry-ext-opentracing-shim/src/opentelemetry/ext/opentracing_shim/__init__.py

```diff
@@ -231,11 +232,16 @@ def start_span(
         # Use the specified parent or the active span if possible. Otherwise,
         # use a `None` parent, which triggers the creation of a new trace.
         parent = child_of.unwrap() if child_of else None
-        span = self._otel_tracer.create_span(operation_name, parent)
 
+        links = ()
         if references:
+            links = []
```

Comment:
Curious, is there a strong reason to create `links` as a tuple and then change to list upon modification?

Reply:
There is not. Cleaned it up